conf/messages

```diff
@@ -774,10 +774,10 @@ previousReturnExistsWithSameCompletionDate.amend.indirectDisposals.title=Indirec
 previousReturnExistsWithSameCompletionDate.amend.indirectDisposals.p1=You’ll need to amend the return that had shares with the same sale of disposal date as the disposal you’re wanting to report now.
 
 previousReturnExistsWithSameCompletionDate.title=You’ll need to change an existing return
-previousReturnExistsWithSameCompletionDate.p1=You already sent a return that had a disposal on <strong class="bold">{0}</strong>. You’ll need to change that return instead of creating a new one.
+previousReturnExistsWithSameCompletionDate.p1=You already have a return with completion date of <strong class="bold">{0}</strong>. You’ll need to change that return instead of creating a new one.
```

Comment:
You already sent a return that had a disposal with a completion date of [date month year].